OrbitLinuxTracing/PerfEventReaders.cpp

```diff
@@ -79,6 +79,9 @@ std::unique_ptr<CallchainSamplePerfEvent> ConsumeCallchainSamplePerfEvent(
       &event->ring_buffer_record.sample_id,
       offsetof(perf_event_callchain_sample, sample_id));
 
+  ring_buffer->ReadValueAtOffset(&event->ring_buffer_record.nr,
```

Comment:
It is confusing that you read this twice. The constructor of `CallchainSamplePerfEvent` should probably set it. Or just use `event->ring_buffer_record.nr = nr` (but I prefer the first).

Reply:
I changed it the way, that the constructor sets the field.